crates/notedeck_columns/src/post.rs

```diff
@@ -119,4 +137,45 @@ impl NewPost {
             .build()
             .expect("expected build to work")
     }
+
+    fn extract_hashtags(content: &str) -> Vec<String> {
+        let mut hashtags = Vec::new();
+        for word in content.split_whitespace() {
+            if word.starts_with('#') && word.len() > 1 {
+                let tag = word[1..].trim_end_matches(|c: char| !c.is_alphanumeric())
+                    .to_string();
+                if !tag.is_empty() {
+                    hashtags.push(tag);
+                }
+            }
+        }
+        hashtags
+    }
+}
```

Comment:
we already do content parsing within nostrdb (urls for r tags, t for hashtags, etc). I just need to expose ndb_parse_content to nostrdb-rs. will do that soon... this is pretty important because it enables uniforms content parsing across all of our apps.

Reply:
I guess this is fine for now though.... we may want to push a Set<String> instead.

They also need to be lowercased in the set

Reply:
> we already do content parsing within nostrdb (urls for r tags, t for hashtags, etc). I just need to expose ndb_parse_content to nostrdb-rs

That makes sense. Out of curiosity what would that code look like exactly once exposed? From what I am seeing this is ultimately pointing to some C code for content parsing.